### PR TITLE
Des 410 edit course view and delete course

### PR DESCRIPTION
**Pull request by Daniielmayo**

### CODE REVIEW

**Review by abdulflorez — APPROVED** [review]

File: components/organisms/Assessment/AssessmentFinished/AssessmentFinished.tsx [hunk 1]
```diff
@@ -44,9 +45,7 @@ export const AssessmentFinished: FC<AssessmentFinishedProps> = ({
     const nextLesson = getNextLesson();
 
     if (nextLesson) {
-      router.push(
-        "/dashboard"
-      );
+      router.push("/dashboard");
     } else {
       console.log("No hay más lecciones disponibles");
```
Comment (DanielCalleCO) [inline components/organisms/Assessment/AssessmentFinished/AssessmentFinished.tsx:50]: review

File: components/organisms/CreateTopicOnCourseAdminPortal/CreateTopicOnCourseAdminPortal.tsx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+      cancelButtonColor: "var(--red)",
+    }).then(async (result: any) => {
+      if (result.isConfirmed) {
+        try {
+          const response = await createTopicOnCourse(
+            nameTopic,
+            userId || "",
+            idCourse.toString()
+          );
+          console.log(response.data.topic);
```
Comment (DanielCalleCO) [inline components/organisms/CreateTopicOnCourseAdminPortal/CreateTopicOnCourseAdminPortal.tsx:57]: review

File: components/organisms/EditCourseTopicManage/EditCourseTopicManage.tsx [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+          }
+
+          Swal.fire({
+            icon: "success",
+            title: "Curso editado",
+            text: "El curso editado correctamente.",
+          });
+          const course = async () => {
+            const data = await fetchCoursesData();
+            console.log(data);
```
Comment (DanielCalleCO) [inline components/organisms/EditCourseTopicManage/EditCourseTopicManage.tsx:109]: review

File: components/organisms/EditCourseTopicManage/EditCourseTopicManage.tsx [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+      showCancelButton: true,
+      confirmButtonColor: "var(--turquoise)",
+      cancelButtonColor: "var(--red)",
+      confirmButtonText: "Sí, eliminar curso",
+      cancelButtonText: "Cancelar",
+    }).then(async (result) => {
+      if (result.isConfirmed) {
+        try {
+          const resposeData = await deleteCourse(userId, courseId);
+          console.log(resposeData);
```
Comment (DanielCalleCO) [inline components/organisms/EditCourseTopicManage/EditCourseTopicManage.tsx:152]: review

File: services/courses/deleteCourse.ts [hunk 5]
```diff
@@ -0,0 +1,40 @@
+export const deleteCourse = async (userId: string, idCourse: string) => {
+  console.log(userId, idCourse);
```
Comment (DanielCalleCO) [inline services/courses/deleteCourse.ts:2]: review

File: services/courses/deleteCourse.ts [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+      {
+        method: "DELETE",
+        headers: {
+          "Content-Type": "application/json",
+        },
+      }
+    );
+
+    const responseData = await response.json();
+    console.log(responseData);
```
Comment (DanielCalleCO) [inline services/courses/deleteCourse.ts:15]: review

File: services/courses/topicOnCourse/createTopicOnCourse.ts [hunk 7]
```diff
@@ -0,0 +1,47 @@
+export const createTopicOnCourse = async (
+  nameTopic: string,
+  userId: string,
+  idCourse: string
+) => {
+  try {
+    console.log("data SERVISES", { nameTopic, userId, idCourse });
```
Comment (DanielCalleCO) [inline services/courses/topicOnCourse/createTopicOnCourse.ts:7]: review. On this, is SERVICES, not SERVI"S"ES

**Review by DanielCalleCO — CHANGES_REQUESTED** [review]

File: components/organisms/Assessment/AssessmentFinished/AssessmentFinished.tsx (hunk 1 above)
Comment (Daniielmayo) [reply components/organisms/Assessment/AssessmentFinished/AssessmentFinished.tsx:50]: @DanielCalleCO  changed for an alert that can give information to the user

File: components/organisms/CreateTopicOnCourseAdminPortal/CreateTopicOnCourseAdminPortal.tsx (hunk 2 above)
Comment (Daniielmayo) [reply components/organisms/CreateTopicOnCourseAdminPortal/CreateTopicOnCourseAdminPortal.tsx:57]: Line deleted

File: components/organisms/EditCourseTopicManage/EditCourseTopicManage.tsx (hunk 3 above)
Comment (Daniielmayo) [reply components/organisms/EditCourseTopicManage/EditCourseTopicManage.tsx:109]: Line deleted

File: components/organisms/EditCourseTopicManage/EditCourseTopicManage.tsx (hunk 4 above)
Comment (Daniielmayo) [reply components/organisms/EditCourseTopicManage/EditCourseTopicManage.tsx:152]: Line deleted

File: services/courses/deleteCourse.ts (hunk 5 above)
Comment (Daniielmayo) [reply services/courses/deleteCourse.ts:2]: Line deleted

File: services/courses/deleteCourse.ts (hunk 6 above)
Comment (Daniielmayo) [reply services/courses/deleteCourse.ts:15]: Line deleted

File: services/courses/topicOnCourse/createTopicOnCourse.ts (hunk 7 above)
Comment (Daniielmayo) [reply services/courses/topicOnCourse/createTopicOnCourse.ts:7]: Line deleted

**Review by DanielCalleCO — APPROVED** [review]

**Review by generaciondesafiante — APPROVED** [review]

**Pull request merged**